language/move-lang/tests/move_check/parser/less_than_space.exp

```diff
@@ -1,12 +1,12 @@
 error: 
 
-   ┌── tests/move_check/parser/less_than_space.move:5:16 ───
+   ┌── tests/move_check/parser/less_than_space.move:5:18 ───
    │
  5 │         if (v< 10) return v;
-   │                ^^ Unexpected '10'
+   │                  ^ Unexpected ')'
    ·
  5 │         if (v< 10) return v;
-   │                -- Expected a type name
+   │                  - Expected '::'
```

Comment:
Hmm. This was never a very clear error message but this makes it worse. I wonder if we could do something better in `parse_module_access` instead of just calling `consume_token` for the "::" after an address. E.G., if there's no "::" report something like: "Expected '::' after address in a module access" (Is "module access" a term that we use in dev docs or error messages? I forget)

Reply:
Updated!